ibm_catalog.json

```diff
@@ -0,0 +1,249 @@
+              "id": "daf2cf91-2384-4205-b7f4-621299a4896d",
+              "version": ">=1.2.2",
+              "flavors": [
+                "standard"
+              ],
+              "catalog_id": "f64499c8-eb50-4985-bf91-29f9e605a433",
+              "optional": true
+            }
+          ],
+          "dependency_version_2": true,
+          "configuration": [
+            {
+              "key": "ibmcloud_api_key",
+              "required": true
+            },
+            {
+              "key": "prefix",
+              "required": true,
+              "description": "The prefix to add to all resources that this solution creates. To not use any prefix value, you can enter the string `__NULL__`."
+            },
+            {
+              "key": "cloud_monitoring_access_key",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_add_cluster_name",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_agent_name",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_agent_namespace",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_agent_tags",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_agent_tolerations",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_enabled",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_endpoint_type",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_instance_region",
+              "display_name": "Region",
+              "required": false,
+              "custom_config": {
+                "type": "vpc_region",
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "config_constraints": {
+                  "generationType": "2"
+                }
+              }
+            },
+            {
+              "key": "cloud_monitoring_metrics_filter",
+              "required": false
+            },
+            {
+              "key": "cloud_monitoring_secret_name",
+              "required": false
+            },
+            {
+              "key": "cluster_config_endpoint_type",
+              "required": false
+            },
+            {
+              "key": "cluster_id",
+              "type": "string",
+              "default_value": "",
+              "display_name": "Cluster",
+              "required": true,
+              "custom_config": {
+                  "type": "cluster_var",
+                  "grouping": "deployment",
+                  "original_grouping": "deployment"
+              }
+            },
+            {
+              "key": "cluster_resource_group_id",
```

Comment:
same as above comment, this is required so should go up the top after `cluster_id`